src/renderer/screens/Preferences.js

```diff
@@ -20,19 +20,12 @@ import Tab from "@mui/material/Tab";
 import Tabs from "@mui/material/Tabs";
 import { GlobalContext } from "@renderer/components/GlobalContext";
 import { PageTitle } from "@renderer/components/PageTitle";
-import PropTypes from "prop-types";
 import React, { useContext } from "react";
 import { useTranslation } from "react-i18next";
 import { UserInterfacePreferences } from "./Preferences/UserInterface";
 import { DevtoolsPreferences } from "./Preferences/Devtools";
 import { MyKeyboardPreferences } from "./Preferences/MyKeyboard";
 
-TabPanel.propTypes = {
```

Comment:
I'd love to leave prop-types around and ideally to use it more.

I'm starting to see the light on typescript and if we can introduce gradual type checking, it'll likely catch a whole slew of errors and make our lives better

Reply:
While I understand the desire, there's 0 point in having prop-types *now*. I'd rather drop it now, so we don't pretend we do anything with it, and reintroduce it when we actually start using it. We could even open an issue to that end, so we don't loose sight.